apps/hash-external-services/docker-compose.temporal.yml

```diff
@@ -140,7 +140,7 @@ services:
       HASH_GRAPH_API_PORT: "${HASH_GRAPH_API_PORT}"
     tmpfs:
       - /tmp
-    read_only: true
+    read_only: false
```

Comment:
@TimDiekmann @CiaranMn just to double check, is there something we have to add to terraform to allow the container to be written to? Or is that already setup? (It can no longer be readonly as the `officeparser` package writes files to the disk temporarily, even if you pass it a `Buffer`)